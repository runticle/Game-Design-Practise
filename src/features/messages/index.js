import React from 'react'

export function Messages(props) {

  return (
    <div style = {{
      float: 'right',
      color: 'white',
      width: '30%',
      height: '80%',
      padding: '5px',
      backgroundColor: "grey"
    }}>
      {props.message}
    </div>
  )

}

export default Messages
