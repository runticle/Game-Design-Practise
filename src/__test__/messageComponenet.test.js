import React from 'react'
import { shallow } from 'enzyme'

import {Messages} from '../features/messages/index'

describe('Messages', () => {
  describe('Messages', () => {
    it('should render correctly', () => {

      const messages = shallow(<Messages />)

      expect(messages).toMatchSnapshot()
    })
  })
})
