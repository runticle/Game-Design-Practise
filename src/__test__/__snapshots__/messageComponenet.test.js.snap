// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages Messages should render correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "grey",
      "color": "white",
      "float": "right",
      "height": "80%",
      "padding": "5px",
      "width": "30%",
    }
  }
/>
`;
